### DISCUSSION TITLE
chezmoi but with custom dashboard-nvim config

### DISCUSSION THREAD

**Discussion by kimusan** [LazyVim/LazyVim#3964] [Q&A]
Hi,
I was happy to see that LazyVim picked up chezmoi support, as I have been using this for a while. 
Right now I have a custom dashboard-nvim config (mainly to change the logo design, but also to actually have a chezmoi config item), and I wanted to adapt the config from https://www.lazyvim.org/extras/util/chezmoi to work in my setup. 

The example however sets the _action_ to "pick_chezmoi" and nothing more. 
If I do the same, nothing happens when the item is selected. I tried with LazyVim.pick("pick_chezmoi") but that also did not seem to work. 

If I add an error to my custom config so it reverts to the default lazyvim config, then it works perfectly. 

Any idea what I am missing here?

PS: previously I used `action = "Telescope chezmoi find_files", `  but I'd like to have it as close to the functionality of vanilla Lazyvim as possible.

**Comment by dpetka2001** [comment]
Well `pick_chezmoi` does `require("telescope").extensions.chezmoi.find_files()` behind the scenes if you're using Telescope, so you might want to try that (though I believe it should be the same as the command you're using). Otherwise just remove your custom action from Dashboard and install Chezmoi Extra and let the Extra handle it.
